### PR TITLE
[WD-156] Responsive dashboard design

### PR DESCRIPTION
**Pull request by christinasugiono**

### CODE REVIEW

File: app/assets/stylesheets/components/_cards.scss [hunk 1]
```diff
@@ -18,7 +18,8 @@
 
 .card-show-wish {
   display: grid;
-  height: 100%;
+  // height: 100%;
```
Comment (Danrod16) [inline app/assets/stylesheets/components/_cards.scss:21]: We can remove comments :)

**Review by Danrod16 — CHANGES_REQUESTED** [review]
Do you have a screenshot of the UI? The code looks nice!

**Comment by christinasugiono** [comment]
image
Screenshot 2023-11-28 at 17 37 35

**Comment by Danrod16** [comment]
Beutiful!

**Pull request merged**